src/components/dialogs/parameters/parameters.js

```diff
@@ -258,7 +255,9 @@ export const TabPanel = (props) => {
             style={{ flexGrow: 1 }}
             {...other}
         >
-            {(value === index || keepState) && <Box p={1}>{children}</Box>}
+            {(value === index || keepState) && (
+                <Box sx={styles.panel}>{children}</Box>
```

Comment:
[The doc](https://mui.com/system/properties/) say that `p` is an abbreviation of `padding`, not margin. Is this wanted ?